src/Microsoft.AspNet.Mvc.ViewFeatures/ViewFeatures/ViewExecutor.cs

```diff
@@ -104,7 +106,7 @@ public class ViewExecutor
             IView view,
             ViewDataDictionary viewData,
             ITempDataDictionary tempData,
-            MediaTypeHeaderValue contentType,
+            MediaTypeHeaderValue actionResultContentType,
```

Comment:
Just for more clarity while reading the code...consider the following piece of code which we try to figure the content type

```
            var resolvedContentType = ResponseContentTypeHelper.GetContentType(
                actionResultContentType,
                response.ContentType,
                DefaultContentType);
```